Sources/Build/Command.compile(ClangModule).swift

```diff
@@ -105,6 +105,7 @@ extension Command {
         var args = module.basicArgs
         args += module.optimizationFlags(conf)
         args += ["-L\(prefix)"]
+        args += module.languageArgs
```

Comment:
This still doesn't completely make sense, we want different language specific arguments for when we link versus when we compile. It only makes sense to pass -lc++ when we link.